css/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 2% 10%;
+  font-family: Poppins, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.menu > li {
+  padding-left: 15px;
+  padding-right: 15px;
+  overflow: none;
+}
+
+.nav-item {
+  text-decoration: none;
+  color: #344563;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.nav-item:hover {
+  color: #6070ff;
+}
+
+.ham-icon {
+  color: #6070ff;
+  font-size: 15px;
+}
+
+#nav-hamburger { display: none; }
+
+#main {
```

Comment:
- [ ] kindly consider aligning the content with flexbox in this section. 
  - In order to lay out the elements on the page you should use flexbox in all 2 sections.